scripts/exgdas_enkf_sfc.sh

```diff
@@ -78,18 +78,32 @@ bPDY=${BDATE:0:8}
 bcyc=${BDATE:8:2}
 
 # Get dimension information based on CASE
-res=${CASE:2:}
+res=${CASE:1}
 JCAP_CASE=$((res*2-2))
 LATB_CASE=$((res*2))
 LONB_CASE=$((res*4))
 
 # Global cycle requires these files
 export FNTSFA=${FNTSFA:-'                  '}
-export FNACNA=${FNACNA:-${COM_OBS}/${OPREFIX}seaice.5min.blend.grb}
-export FNSNOA=${FNSNOA:-${COM_OBS}/${OPREFIX}snogrb_t${JCAP_CASE}.${LONB_CASE}.${LATB_CASE}}
-[[ ! -f $FNSNOA ]] && export FNSNOA="${COM_OBS}/${OPREFIX}snogrb_t1534.3072.1536"
-FNSNOG=${FNSNOG:-${COM_OBS_PREV}/${GPREFIX}snogrb_t${JCAP_CASE}.${LONB_CASE}.${LATB_CASE}}
-[[ ! -f $FNSNOG ]] && FNSNOG="${COM_OBS_PREV}/${GPREFIX}snogrb_t1534.3072.1536"
+export FNACNA=${FNACNA:-${COMIN_OBS}/${OPREFIX}seaice.5min.blend.grb}
+export FNSNOA=${FNSNOA:-${COMIN_OBS}/${OPREFIX}snogrb_t${JCAP_CASE}.${LONB_CASE}.${LATB_CASE}}
+# Check if resolution specific FNSNOA exists, if not use t1534 version
+[[ ! -f ${FNSNOA} ]] && export FNSNOA="${COMIN_OBS}/${OPREFIX}snogrb_t1534.3072.1536"
+if [[ ! -f ${FNSNOA} ]]; then
+  echo "FATAL ERROR: Current cycle snow file ${FNSNOA} is missing. Exiting."
+  exit 1
+else
+  echo "INFO: Current cycle snow file is ${FNSNOA}"
+fi
+export FNSNOG=${FNSNOG:-${COMIN_OBS_PREV}/${GPREFIX}snogrb_t${JCAP_CASE}.${LONB_CASE}.${LATB_CASE}}
+# Check if resolution specific FNSNOG exists, if not use t1534 version
+[[ ! -f ${FNSNOG} ]] && export FNSNOG="${COMIN_OBS_PREV}/${GPREFIX}snogrb_t1534.3072.1536"
+if [[ ! -f ${FNSNOG} ]]; then
+  echo "FATAL ERROR: Previous cycle snow file ${FNSNOG} is missing. Exiting."
```

Comment:
```suggestion
if [[ ! -f ${FNSNOA} ]] && [[ ! -f ${FNSNOG} ]]; then
  echo "FATAL ERROR: Both current (${FNSNOA}) and previous cycle (${FNSNOG}) snow files are missing."
```

Reply:
@GeorgeGayno-NOAA is this correct, or should it always fail if the snow guess is missing, even if the current snow is present?